src/components/BuilderComponents/CustomLinkBoxes.js

```diff
@@ -42,33 +17,40 @@ const CustomLinkBoxes = ({ data }) => {
       <ul className="custom-link-boxes__list">
         {!!list &&
           list.map((item, i) => (
-            <CustomLinkBox
-              key={`custom-link-box-${i}`}
-              link={item.content}
-              mdContent={item.mdContent}
-              boxColor={item.fgColor}
-              textAlign={item.textAlign}
-              textColor={item.textColor}
-            />
+            <li>
+              <CustomLinkBox
+                key={`custom-link-box-${i}`}
+                link={item.content}
```

Comment:
think this mismatch is what's giving you an issue per last comment

Reply:
I think you're right, it's hard to track down. I wanted to rename them so they would fit their function in the component better, but now that the documentation explains what they're are it should get the job done. I'll make it consistent.